### ISSUE TITLE
cursorColor, allow color names?

### ISSUE THREAD

**Issue by vefatica** [microsoft/terminal#7522]
I can't do this

"cursorColor": "yellow"

AFAICT, to get a "yellow" cursor I must (look up and) hard code the "yellow" value from the current theme.

Is there a better way?  If not I suggest that you allow color names for "cursorColor"

And what's the difference between "#rgb" and "#rrggbb"?

**Comment by DHowett** [comment]
I was sure that we had a feature request for this already, but it looks like we don't! Thanks!

The difference between `#rgb` and `#rrggbb` is that `#rgb` is shorter.

**Comment by DHowett** [comment]
Link: #3327

**Comment by DHowett** [comment]
Link: #5775

**Comment by vefatica** [comment]
I searched too (but not as hard as you).

And/or ... allow an integer index in the palette, as in

"cursorColor": 3 // "yellow"

**Comment by vefatica** [comment]
_The difference between #rgb and #rrggbb is that #rgb is shorter._

That's obvious.  :-)  Is there any difference in their use?  Is there any alternative to "#123456"?

**Comment by DHowett** [comment]
Nah. I believe the `#rgb` shorthand comes from CSS, and as such is very convenient for _web color grading_. You specify one nybble for each color, and that nybble is copied over the one you didn't specify: `#c12` becomes `#cc1122` and `#fff` becomes `#ffffff`.

There's no shorthand for `#123456` (nor can there ever be?) because there's no way to specify the same amount of information _by specifying less information_ 😉

**Comment by vefatica** [comment]
_You specify one nybble for each color, and that nybble is copied over the one you didn't specify: #c12 becomes #cc1122 and #fff becomes #ffffff._

Thanks for the explanation.  That's odd.  I never saw it before.
